docs/source/docs/troubleshooting/unix-commands.md

```diff
@@ -0,0 +1,106 @@
+Useful Unix Commands
+====================
+
+SSH
+---
+
+[SSH (Secure Shell)](https://www.mankier.com/1/ssh) is used to securely connect from a local to a remote system (ex. from a laptop to a coprocessor). Unlike other commands on this page, ssh is not Unix specific and can be done on Windows and MacOS from their respective terminals.
+
+:::{note}
+You may see a warning similar to `The authenticity of host 'xxx' can't be established...` or `WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!`, in most cases this can be safely ignored if you have confirmed that you are connecting to the correct host over a secure connection, and the fingerprint will change when your operating system is reinstalled or PhotonVision's coprocessor image is re-flashed. You can read more about it [here](https://superuser.com/questions/421997/what-is-a-ssh-key-fingerprint-and-how-is-it-generated)
+:::
+
+Example:
+
+:::{code-block}
+ssh username@hostname
+:::
+
+ip
+--
+
+Run [ip address](https://www.mankier.com/8/ip) with your coprocessor connected to a monitor in order to see its IP address and other network configuration information.
+
+
+SCP
+---
+
+[SCP (Secure Copy)](https://www.mankier.com/1/scp) is used to securely transfer files between local and remote systems.
+
+Example:
+
+:::{code-block}
+scp [file] username@hostname:/path/to/destination
+:::
+
+SFTP
+---
+
+[SFTP (SSH File Transfer Protocol)](https://www.mankier.com/1/sftp#) is another option for transferring files between local and remote systems.
```

Comment:
Do we want to suggest GUI alternatives? And we should mention file ownership pitfalls? (all of /opt/photonvision is owned by root ex)